### PR TITLE
Update README.doc (noref)

### PR DESCRIPTION
**Pull request by csymons-suse**
- Add commit requirements--line length, title format, 
  message body required (no one line commits).
- Correct typo.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -69,14 +69,17 @@ Contributing
 
 Thank you for contributing to this repo. Please adhere to the following guidelines when creating a pull request:
 
-. Make your pull request against the _master_ branch (not develio) if you are contributing to the most recent release (currently
+. Make your pull request against the _master_ branch (not develop) if you are contributing to the most recent release (currently
```
Comment (sknorr) [inline README.adoc]: Oof. Thanks for noticing.

File: README.adoc [hunk 2]
```diff
@@ -69,14 +69,17 @@ Contributing
...
   Cloud 10). The _master_ branch is protected and cannot be merged without review.
 
 . If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also 
   protected and cannot be merged without review.
 
 . Your contributions can be in ASCIIdoc `.adoc` or DocBook XML `.xml`.
 
+. Commit messages must have line lengths of 80 characters or less. Commit body content is required. The commit title must
+  match regex (^(?!SOC).*\((((bsc#|SOC-)[0-9]+(,\s)?)+|trivial|typo|noref)\)$)
```
Comment (sknorr) [inline README.adoc]: Please explain in more human-friendly terms:
* Subject line must not start with "SOC"
* At the end of the subject line, there must be a bug reference or explanation for a missing bug reference:
  * bug references: "(bsc#0123456)" or "(SOC-0123456)"
  * multiple bug references: "(bsc#0123456)" or "(SOC-0123456)"
  * the values "(trivial)", "(typo)", or "(noref)", no combinations
    allowed

Comment (sknorr) [reply README.adoc]: @rsalevsky One thing I am wondering -- aren't you using SCRD- for Jira references?
(Also Jira references in the rest of SUSE are actually written with jsc# infront, i.e. jsc#PRJ-0123.)

Comment (csymons-suse) [reply README.adoc]: changed. Please check though, not sure about multiple references. Debated about quotes or not.

File: README.adoc [hunk 3]
```diff
@@ -69,26 +69,36 @@ Contributing
...
 . Your contributions can be in ASCIIdoc `.adoc` or DocBook XML `.xml`.
 
+. Commit messages must conform to the following:
+.. Line lengths must be 80 characters or less.
+.. Commit body content is required.
+.. The subject line cannot start with "SOC".
+.. At the end of the subject line, there must be a bug reference or an
+   explanation for a missing bug reference:
+*** Bugzilla or Jira references: (bsc#0123456) or (SOC-0123456)
+*** Multiple Bugzilla or Jira references: (bsc#0123456),(SOC-0123456)
```
Comment (sknorr) [inline README.adoc]: Just saw that my original comment on multiple references read thus:

> multiple bug references: "(bsc#0123456)" or "(SOC-0123456)"

I should have said:

> multiple bug references: e.g. "(bsc#0123456, SOC-0123456)"

Sorry.

And yes, multiple references should work, according to the regex. What does not work is combinations of multiple non-references, e.g.: "(trivial, typo)" does not work.

File: README.adoc [hunk 4]
```diff
@@ -69,26 +69,36 @@ Contributing
...
-. If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also 
+. If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also
   protected and cannot be merged without review.
 
 . Your contributions can be in ASCIIdoc `.adoc` or DocBook XML `.xml`.
 
+. Commit messages must conform to the following:
+.. Line lengths must be 80 characters or less.
+.. Commit body content is required.
+.. The subject line cannot start with "SOC".
```
Comment (sknorr) [inline README.adoc]: This is definitely in the regex but fwiw, I have no clue how this rule came to be.

**Pull request merged**